scripts/build/build-utils.ps1

```diff
@@ -147,6 +147,15 @@ function Invoke-UnitTests([string]$binPath, [bool]$failsIfNotTest) {
 
     & (Get-VsTestPath) $testFiles /Parallel /Enablecodecoverage /InIsolation /Logger:trx /TestAdapterPath:$testDirs
     Test-ExitCode "ERROR: Unit Tests execution FAILED."
+
```

Comment:
I might be useful to add log like `Write-Header "Running unit tests .NET Core 3.1"` to report the stage. Also update the message for the first run on line 131